mono/mini/method-to-ir.c

```diff
@@ -2679,6 +2679,7 @@ mono_emit_method_call (MonoCompile *cfg, MonoMethod *method, MonoInst **args, Mo
 MonoInst*
 mono_emit_native_call (MonoCompile *cfg, gconstpointer func, MonoMethodSignature *sig,
 					   MonoInst **args)
+// FIXME tailcall?
 {
```

Comment:
These are never going to be tailcalls.

Reply:
Because we have to save lmf?  I can remove/reduce the comments.

Reply:
Because these types of calls are generated by the JIT, and we don't use tail calls ourselves.